apps/api/src/domain/models/term/commands/create-term/multilingual-text-auto-indexer.event-handler.ts

```diff
@@ -0,0 +1,7 @@
+import { ICoscradEvent, ICoscradEventHandler } from '../../../../../domain/common';
+
+export class MultilingualTextAutoIndexer implements ICoscradEventHandler {
+    handle(event: ICoscradEvent): Promise<void> {
```

Comment:
Oh yea. This is what this PR is getting at. This is going to be really powerful!

Reply:
BTW, is there a test for this?

Reply:
I've decided to leave this and other applications for future PRs. Now that we've updated the decorator, it opens the option to do this.

I do have a second design in mind where we use the view-model schema and index directly on write. The problem with this is that we have to be able to run a custom indexer to leverage non-English NLP and we do not want to try to deploy that logic to Arango's JS run-time. At any rate, let's leave this as future-scoped.